packages/embed-playground/package.json

```diff
@@ -42,8 +42,8 @@
     "@looker/embed-components": "*",
     "@looker/extension-utils": "^0.1.21",
     "@looker/redux": "^0.0.1",
-    "@looker/sdk": "^23.2.0",
-    "@looker/sdk-rtl": "^21.6.0",
+    "@looker/sdk": "^23.14.1",
+    "@looker/sdk-rtl": "^21.6.1",
```

Comment:
If we set these to `*` does release-please update them for publishing? Or is this an extra lint/package dependency step we're going to need to add to our generation/publishing step? @treshugart would like your thoughts here

Reply:
To be honest I dont know. I'm just syncing the packages.

Reply:
Understood. That's why I summoned Trey 😸

Reply:
This is *not* a blocker for your PR, which I've approved.